packages/backblaze-b2/src/BackblazeB2Provider.ts

```diff
@@ -92,11 +92,10 @@ class BackblazeB2Provider extends StorageProvider {
      * @async
      */
     isContainer(container: string): Promise<boolean> {
-        // There's no method in the B2 APIs to get a single bucket, so list all buckets and look for the one we're interested in
-        return this.listContainers()
-            .then((list) => {
-                return list.indexOf(container) >= 0
-            })
+        return this._client.getBucket({bucketName: container})
+            .then(result => !!result)
+            // Treat exceptions as not found
+            .catch(() => false)
```

Comment:
Please don't treat all exceptions as "not found". Some exceptions should be allowed to bubble up.

Reply:
I agree, I thought I was following your example.
I only did that, because of what I saw here:
https://github.com/ItalyPaleAle/SMCloudStore/blob/master/packages/generic-s3/src/GenericS3Provider.ts#L60

Should I change it in both places?

Reply:
FYI I fixed it in the other provider